docs/paictl/paictl-manual.md

```diff
@@ -185,16 +187,17 @@ python paictl.py cluster k8s-clean -p /path/to/cluster-configuration/dir
 
 - Stop Kubernetes in the specified cluster.
 
-### Generate cluster-configuration template files from a machine list <a name="Cluster_Conf_Generate"></a>
+### Setup KUBECONFIG and install kubectl in the environment <a name="Cluster_env_setup"></a>
 
 ```
-python paictl.py config generate -p /path/to/machinelist.csv
+python paictl.py config k8s-set-env [ -p /path/to/cluster/configuration ]
 ```
 
-- The machine list should be provided in CSV format.
-- All configuration files of cluster-configuration will be generated to the local folder, and then be used for bootstrapping the whole cluster.
-- By default, in the generated configuration, a single-master Kubernetes is configured by default.
-- Advanced users or developers can fine-tune the content of the generated configuration files according to specific environments.
+- Install kubectl in the deployment box.
+- Configure KUBECONFIG for kubectl or other kubernetes' client.
+- The args ```-p /path/to/cluster/configuration``` is optional. If admin don't pass this args, paictl will ask admin to input the necessary infomation to generate KUBECONFIG.
+- When in a clean dev-box, please run this command firstly.
```

Comment:
Will it prompt hints if user forget to run it first?

Reply:
If user forget to setup environment, all command will failed. I will add information in the future.

Reply:
Could we provide a command displaying the currently config?
For example `python paictl.py config info`.

So user can double check which cluster they are working on.

Reply:
I have provided a command ```paictl config get``` to get the latest configuration